AndroidWeek1/app/build.gradle

```
@@ -0,0 +1,59 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-kapt'
+}
+
+android {
+    compileSdk 30
+    dataBinding {
+        enabled true
+    }
```

안드로이드 스튜디오 4.0 버전 이상부터는 Gradle에서
```suggestion
buildFeatures {
   dataBinding true
}
```
이런식으로 바인딩을 선언합니다~!!